mods/mtg/ctf_changes/init.lua

```diff
@@ -68,3 +68,101 @@ minetest.register_on_mods_loaded(function()
+	drawtype = "allfaces_optional",
+	waving = 1,
+	tiles = {"default_leaves.png"},
+	special_tiles = {"default_leaves_simple.png"},
+	paramtype = "light",
+	is_ground_content = false,
+	groups = {snappy = 3, leafdecay = 3, flammable = 2, leaves = 1},
+	sounds = default.node_sound_leaves_defaults(),
+
+	after_place_node = after_place_leaves,
+})
+
+minetest.register_node(":default:jungleleaves", {
+	description = "Jungle Tree Leaves",
+	drawtype = "allfaces_optional",
+	waving = 1,
+	tiles = {"default_jungleleaves.png"},
+	special_tiles = {"default_jungleleaves_simple.png"},
+	paramtype = "light",
+	is_ground_content = false,
+	groups = {snappy = 3, leafdecay = 3, flammable = 2, leaves = 1},
+	sounds = default.node_sound_leaves_defaults(),
+
+	after_place_node = after_place_leaves,
+})
+
+minetest.register_node(":default:acacia_leaves", {
+	description = "Acacia Tree Leaves",
+	drawtype = "allfaces_optional",
+	tiles = {"default_acacia_leaves.png"},
+	special_tiles = {"default_acacia_leaves_simple.png"},
+	waving = 1,
+	paramtype = "light",
+	is_ground_content = false,
+	groups = {snappy = 3, leafdecay = 3, flammable = 2, leaves = 1},
+	sounds = default.node_sound_leaves_defaults(),
+
+	after_place_node = after_place_leaves,
+})
+
+minetest.register_node(":default:aspen_leaves", {
+	description = "Aspen Tree Leaves",
+	drawtype = "allfaces_optional",
+	tiles = {"default_aspen_leaves.png"},
+	waving = 1,
+	paramtype = "light",
+	is_ground_content = false,
+	groups = {snappy = 3, leafdecay = 3, flammable = 2, leaves = 1},
+	sounds = default.node_sound_leaves_defaults(),
+
+	after_place_node = after_place_leaves,
+})
+
+minetest.register_node(":default:bush_leaves", {
+	description = "Bush Leaves",
+	drawtype = "allfaces_optional",
+	tiles = {"default_leaves_simple.png"},
+	paramtype = "light",
+	groups = {snappy = 3, flammable = 2, leaves = 1},
+	sounds = default.node_sound_leaves_defaults(),
+
+	after_place_node = after_place_leaves,
+})
+
+minetest.register_node(":default:blueberry_bush_leaves", {
+	description = "Blueberry Bush Leaves",
+	drawtype = "allfaces_optional",
+	tiles = {"default_blueberry_bush_leaves.png"},
+	paramtype = "light",
+	groups = {snappy = 3, flammable = 2, leaves = 1},
+	drop = "default:blueberry_bush_leaves",
+	sounds = default.node_sound_leaves_defaults(),
+	on_timer = function(pos, elapsed)
+		if minetest.get_node_light(pos) < 11 then
+			minetest.get_node_timer(pos):start(200)
+		else
+			minetest.set_node(pos, {name = "default:blueberry_bush_leaves_with_berries"})
+		end
+	end,
+
+	after_place_node = after_place_leaves,
+})
+
+minetest.register_node(":default:acacia_bush_leaves", {
+	description = "Acacia Bush Leaves",
+	drawtype = "allfaces_optional",
+	tiles = {"default_acacia_leaves_simple.png"},
+	paramtype = "light",
+	groups = {snappy = 3, flammable = 2, leaves = 1},
+	drop = "default:acacia_bush_leaves",
+	sounds = default.node_sound_leaves_defaults(),
+
+	after_place_node = after_place_leaves,
+})
```

Comment:
Use a list of the nodes you want to override (`{"node:name1", "node:name2", ...}`) and `minetest.override_item()` in a `for` loop that loops through that list